tests/introspection/Mac/MacApiProtocolTest.cs

```diff
@@ -171,6 +171,13 @@ protected override bool Skip (Type type, string protocolName)
 					return true; 
 				}
 				break;
+			case "NSCandidateListTouchBarItemDelegate":
```

Comment:
That looks like a category (`NSTextView_TouchBar`) that implements a protocol (`NSCandidateListTouchBarItemDelegate`).

First time I've ever seen that...